src/ows/src/main/java/org/geoserver/ows/Request.java

```diff
@@ -48,6 +48,8 @@
     /** OWS request (ie operation) combined with service and version */
     protected String request;
 
+    protected String postRequestEementName;
```

Comment:
Eement?

It's used in a lots of places, should be renamed with a refactor and make sure it's fixed everywhere.

Reply:
https://github.com/geoserver/geoserver/pull/4804/commits/86998b5ed3b342f44e2387a50b28077b37d3b16e#diff-be2ee09ab4b472e5452386650f99cd7c3381ff0c97c98396b39cb048212b7b01R308-R325